### ISSUE TITLE
Failure with dev version of testthat

### ISSUE THREAD

**Issue by hadley** [dgrtwo/broom#132]
```R
Last 13 lines of output:
  3: as.expectation(exp, ..., srcref = srcref)
  4: identical(as.vector(object), TRUE)
  5: as.vector(object)
  6: augmented$disp
  7: `$.tbl_df`(augmented, disp)
  8: stop("Unknown column '", i, "'", call. = FALSE)
  
  testthat results ================================================================
  OK: 490 SKIPPED: 0 FAILED: 1
  1. Error: rowwise tidiers can be applied to sub-models (@test-rowwise.R#21) 
  
  Error: testthat unit tests failed
  Execution halted
```

This might be because tidyr now produces `tbl_df` and this triggers the stricter column access checks in tibble.

**Comment by hadley** [comment]
Hmmm, maybe I misdiagnosed the problem - tidyr hasn't changed, so maybe it's purely the tibble issue?

**Comment by hadley** [comment]
Oh but tidyr now attaches tibble, which has the stricter checks. That's likely to be the cause, although I'm not sure how `[.tbl_df` gets dispatched when there are two packages that provide it

cc @krlmlr

**Comment by krlmlr** [comment]
Can't replicate with 3b9a04bb62dc5, hadley/testthat@0ae034efb5560b, hadley/tidyr@32d9dd2ab8c25bd1, and CRAN tibble nor hadley/tibble@19235d21f4322b7 on Ubuntu 15.10. I think I also tested CRAN tidyr.

**Comment by dgrtwo** [comment]
This should have been fixed with a0ada0a17412398738aa50dbba59c4dfc81fb8fb (and can't replicate on new dev version)

broom 0.4.1 including this change will be submitted to CRAN by tonight
